src/test/lib/statistics/statistics_objects/min_max_filter_test.cpp

```diff
@@ -48,6 +48,63 @@ class MinMaxFilterTest<pmr_string> : public BaseTest {
   pmr_string _before_range, _min_value, _max_value, _after_range, _in_between, _in_between2;
 };
 
+class MinMaxFilterTestLike : public BaseTest {
+ protected:
+  void SetUp() override {
```

Comment:
Why use the `SetUp` method?

Reply:
I thought it would be easier to read. If you want I can remove the `SetUp`.

Reply:
It's unnecessary and allows reducing the code. I would remove it.